src/QueryBuilder.php

```diff
@@ -115,7 +116,7 @@ private function buildConditions(string $index, array $blocks): array
         $conditions = $this->buildConditionsByType($index, $blocks);
 
         if ($this->includeAccessData($index)) {
-            $conditions[] = $this->buildAccessConditions();
+            $conditions[] = $accessConditions;
```

Comment:
`$accessConditions` is not defined here, maybe this change is committed by mistake?

Reply:
True, fixed :) Thanks!